lib/server/dsu_server.dart

```diff
@@ -73,8 +75,6 @@ class DSUServer {
       }
 
       clients[address] = port;
-
-      this.report(slots[0]);
```

Comment:
CEMU sends a lot of these incoming data request, [response is not required](https://v1993.github.io/cemuhook-protocol/#:~:text=When%20you%20get%20this%20type%20of%20message%2C%20no%20instant%20response%20is%20required.).